libraries/engage/locations/components/StockInfo/StockInfoStoreName.jsx

```diff
@@ -1,36 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { capitalize } from '../../../core';
```

Comment:
Could do you please also remove this helper `capitalize`. It was introduced i August, when the first ropis engage stuff was building. I dont see use cases for js capitalization. Pure CSS can be sued instead

Reply:
Removed